src/main/scala/is/hail/HailContext.scala

```diff
@@ -511,15 +511,17 @@ class HailContext private(val sc: SparkContext,
     forceBGZ: Boolean = false,
     headerFile: Option[String] = None,
     nPartitions: Option[Int] = None,
-    sitesOnly: Boolean = false): GenericDataset = {
-    importVCFsGeneric(List(file), force, forceBGZ, headerFile, nPartitions, sitesOnly)
+    sitesOnly: Boolean = false,
+    callFields: Option[Set[String]] = None): GenericDataset = {
```

Comment:
What's the difference between None and Set.empty[String]?  If nothing, just make it `callFields: Set[String] = Set.empty`.